.gitignore

```diff
@@ -63,3 +63,6 @@ coverage.xml
 
 # Sphinx documentation
 doc/_build/
+.vscode/
+Pipfile
```

Comment:
Pipfile and Pipfile.lock files are newly added in this PR, which seems at odds with having .gitignore entries.  Are the files supposed to be in the PR?